config/assisted-service/postgres-secret.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  labels:
+    app: postgres
+  name: assisted-installer-rds
+  namespace: assisted-installer
+stringData:
+  db.host: postgres
+  db.name: installer
+  db.password: admin
```

Comment:
Can this be randomly generated somehow?
Or is this not the actual password that will get used?

Reply:
Added an init container to randomly generate the password.